### ISSUE TITLE
Feature Request - hybrid analysis to use more than one module

### ISSUE THREAD

**Issue by shadowk29** [usnistgov/mosaic#63]
CUSUM works well for long events but does a poor job on short ones. SRA does very well with short events, but is not usually suited to long events that might have multiple levels. 

Ideally, a user could set a number of data points - events which are longer than this are fed to CUSUM or MSA, and events that are shorter are fed to SRA. This gives the speed of CUSUM for multilevel events and the accuracy of SRA for short ones.

**Comment by abalijepalli** [comment]
The logic for such a switch is easy to implement. It may however require us to rethink the DB format. Currently SRA and CUSUM have different DB tables. It is however possible to force SRA to use the CUSUM/MSA output format.

**Comment by shadowk29** [comment]
Standardizing the database format across the analysis methods might not be a bad idea anyway, for the sake of easy scripting/analysis of the output.

**Comment by abalijepalli** [comment]
I agree. In the short term it will break a lot of pre-existing analysis, so it will require some thought. Perhaps some translation layer that upgrades old databases will be needed.
